src/gluonts/transform/feature.py

```diff
@@ -249,18 +247,15 @@ class AddConstFeature(MapTransformation):
     the T-dimension is defined as the sum of the `pred_length` parameter and
     the length of a time series specified by the `target_field`.
 
-    If `is_train=True` the feature matrix has the same length as the `target` field.
-    If `is_train=False` the feature matrix has length len(target) + pred_length
-
     Parameters
     ----------
     output_field
         Field name for output.
     target_field
         Field containing the target array. The length of this array will be used.
     pred_length
-        Prediction length (this is necessary since
-        features have to be available in the future)
+        Prediction length (this is necessary since features have to be available
```

Comment:
So to clarify: The change in behavior is that we ALWAYS generate the features for additional `prediction_length` steps, right?

Reply:
Reverted